src/mbgl/renderer/layers/render_location_indicator_layer.hpp

```diff
@@ -23,7 +23,7 @@ class RenderLocationIndicatorLayer final : public RenderLayer {
     void render(PaintParameters &) override;
 
     bool contextDestroyed = false;
-    RenderLocationIndicatorImpl *renderImpl = nullptr;
+    std::unique_ptr<RenderLocationIndicatorImpl> renderImpl;
```

Comment:
how this `RenderLocationIndicatorImpl`  instance was managed before? was it leaking?

Reply:
it was probably leaking, yes..

Reply:
Curious how the CI did not detected this. It should have been visible on the Valgrind bot (but at the moment it is not failing). Also clang-tidy has a check for not using `new` like this that is currently disabled.